esmf_regrid/experimental/io.py

```diff
@@ -52,11 +52,18 @@ def save_regridder(rg, filename):
     regridder_type = rg.__class__.__name__
     if regridder_type == "GridToMeshESMFRegridder":
         src_grid = (rg.grid_y, rg.grid_x)
```

Comment:
I'd suggest that instead of building this and then indexing it, we could just refer to `rg.grid_y` and `rg.grid_x` as we go through this code - I think it will be easier to follow what's going on

Reply:
My thinking here is that this might help to keep things clear when we add saving for grid-to-grid regridders. In this case there will be both source and target grids and a potential to reuse this code. Woth that said, I suppose there is room to refactor this code already to reduce code reuse so I might try some refactoring.